backend/src/main/java/heartbeat/service/pipeline/buildkite/BuildKiteService.java

```diff
@@ -76,15 +77,13 @@ public BuildKiteResponse fetchPipelineInfo(PipelineParam pipelineParam) {
 			log.error("[BuildKite] Failed when call BuildKite", e);
 			throw new RequestFailedException(e);
 		}
-		catch (NoPermissionException e) {
-			throw new RequestFailedException(403, e.getMessage());
-		}
 	}
 
-	private void verifyToken(BuildKiteTokenInfo buildKiteTokenInfo) throws NoPermissionException {
+	private void verifyToken(BuildKiteTokenInfo buildKiteTokenInfo) {
 		for (String permission : permissions) {
 			if (!buildKiteTokenInfo.getScopes().contains(permission)) {
-				throw new NoPermissionException("Permission deny!");
+				log.error("[BuildKite] Failed when call BuildKite Permission deny!");
```

Comment:
change the error log to 
```java
log.error("Failed to call BuildKite, because of insufficient permission, current permissions: {}", buildKiteTokenInfo.getScopes())
```

Reply:
done